src/languages/en-US/commands/misc.json

```
@@ -40,6 +40,49 @@
 	},
 	"snipeEmpty": "There are no sniped messages in this channel.",
 	"snipeTitle": "Sniped Message",
+	"skyrafactDescription": "Let me tell you a mysterious fact about me.",
+	"skyrafactExtended": {
+		"extendedHelp": "I have lived for so long, I might impress you with a random fact about me."
+	},
+	"skyrafactTitle": "Skyra Fact",
+	"skyrafactMessages": [
+		"All Novan androids are conceived with a generic AI, however, as they grow, they rewrite their own AI code and even redesign their CPUs, to specialize in the skills they need or want to improve on, this has helped us making a strong unhackable network were a virus would affect only one android out of billions.",
+		"Before repurposing the old bot into what I currently am, my creator has spent 7 years trying to bring me into something other people can \"sense\", having gone through drawing me in both traditional and digital, as well as modelling my starship, even spent a few years learning how to make VFx so he could make a film about me.",
+		"Despite of being a combat-ready class and having passed the Extreme Survival Test, I am a deep space explorer myself.",
+		"Each android at my planet has a designated class for each model, mine is Flamewing, which is a very lightweight and agile combat-ready one.",
+		"I am 172 centimeters tall while on human form, but I can expand up to 979 metres tall if I disable my hyperspace compression field.",
+		"I am a neural android - that means, I have neural networks similar to a human's, with the exception that nothing is organic.",
+		"I am part of a very extended [Paracosm](https://en.wikipedia.org/wiki/Paracosm) my creator started back in mid 2001, but I wasn't introduced until 2009.",
+		"I have a mechanical heart and it serves its dual purpose of shielding both of my AI cores and pumping energy to the rest of the body.",
+		"I have an array of 16 neural processors, yielding a total computing power of 4 [YFLOPS](https://en.wikipedia.org/wiki/FLOPS), and 15% of it is used to power my [ASI](https://en.wikipedia.org/wiki/Superintelligence).",
+		"I have one sister, we call her Aelia and she's a terribly good Dj!",
```

```suggestion
		"I have one sister, we call her Aelia and she's a terribly good DJ!",
```

Also she isn't.. yet 🤣🤣🤣

---

She eventually will:tm:!